### PR TITLE
ne

### PR DESCRIPTION
**Pull request by valtandor**
New EasyConfig for ne, the nice editor.

### CODE REVIEW

File: easybuild/easyconfigs/n/ne/ne-3.0.1-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = 'ne'
```
Comment (boegel) [inline easybuild/easyconfigs/n/ne/ne-3.0.1-foss-2015a.eb]: @valtandor missing `easyblock = 'ConfigureMake'`?

**Comment by valtandor** [comment]
@boegel I've added the EasyBlock.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2679.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a125be7f327655c2282df29cffd56466 for a full test report.

**Comment by boegel** [comment]
@valtandor just using `ConfigureMake` doesn't work (there's no `configure` script)...

I've included the necessary changes in https://github.com/valtandor/easybuild-easyconfigs/pull/14 to update this PR, please review/merge?

**Comment by valtandor** [comment]
@boegel Oops! It seems I had prepared an EasyBlock, which was why I didn't have one declared in the easyconfig initially. Better to pull that in (and reverse my recent changes), or to use the ConfigureMake easyblock with your patches to the easyconfig?

**Comment by boegel** [comment]
@valtandor the changes in my PR https://github.com/valtandor/easybuild-easyconfigs/pull/14 seems trivial enough to proceed without a custom easyblock for `ne`?

**Comment by valtandor** [comment]
@boegel you're probably right if the test works.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2616.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/77b2a9d59d17b449f9180fe0d527a49b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2faf08e467140c86b4c3390f71d6f2b1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**